Update docker-compose.yml

rebeccaSimmonds19 opened the pull request:
Removed comment and changed the postgres port to only use 5432 in the docker-compose.yml

Tested on MacBook Pro 2023

Removed error when connecting to the REST server

"Caused by: org.postgresql.util.PSQLException: Connection to postgres:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections."

rebeccaSimmonds19 committed:
Update docker-compose.yml

Removed comment and changed port to only use 5432

Signed-off-by: Rebecca Whitworth <rmsimmonds1919@gmail.com>

rebeccaSimmonds19 committed:
Update docker-compose.yml

Removed comment and changed port to only use 5432

kiersten-stokes reviewed (commented):
Thank you so much for this! I have one comment below.

Also IBM public repo ha the `DCO` check by default, essentially meaning that commits have to be signed. If you click on the `DCO` test, it will give you some easy instructions to do so 🙂

rkurniawati committed:
Minor fixes to mc and rest containers (#8)

Signed-off-by: Ruth Kurniawati <ruth.kurniawati@gmail.com>
Signed-off-by: Rebecca Whitworth <rmsimmonds1919@gmail.com>

rebeccaSimmonds19 committed:
Merge branch 'IBM:main' into main

rebeccaSimmonds19 committed:
remove ports for docker-compose.yaml

Remove port specification for postgres as it is not needed to specify for local or remote containers.

Signed-off-by: Rebecca Whitworth <rmsimmonds1919@gmail.com>

rebeccaSimmonds19 committed:
remove ports for docker-compose.yaml

Remove port specification for postgres as it is not needed to specify for local or remote containers.

rebeccaSimmonds19 commented on conf/docker-compose.yml:
```diff
@@ -43,7 +43,7 @@ services:
     hostname: postgres
     ports:
-      - 5433:5432  # todo change this back to 5432
+      - 5432
```
I have removed the ports in the last commit as they are not needed. Thanks

kiersten-stokes commented:
This looks great! Thank you so much @rebeccaSimmonds19 . Only thing missing is the commit signoff - it's pretty easy to fix, they give exact copy/paste instructions [here](https://github.com/IBM/presto-iceberg-lab/pull/9/checks?check_run_id=48878361403). Then I'll go ahead and merge 🙂

rebeccaSimmonds19 committed:
Merge branch 'main' of https://github.com/rebeccaSimmonds19/presto-iceberg-lab

rebeccaSimmonds19 commented:
Hey I used the rebase suggested but I think it may have rebased too far?

kiersten-stokes commented:
@rebeccaSimmonds19 so sorry, this fell off my plate for a bit, but LGTM! Thank you for the fix!
